targets/ChibiOS/RP_PICO_W_RP2040/Target_BlockStorage_RP2040FlashDriver.c

```
@@ -0,0 +1,159 @@
+
+    return true;
+}
+
+DeviceBlockInfo *RP2040FlashDriver_GetDeviceInfo(void *context)
+{
+    MEMORY_MAPPED_NOR_BLOCK_CONFIG *config = context;
+
+    return config->BlockConfig.BlockDeviceInformation;
+}
+
+bool RP2040FlashDriver_Read(void *context, ByteAddress startAddress, unsigned int numBytes, unsigned char *buffer)
```

Cppcheck (2.20.0)

[style] The function 'RP2040FlashDriver_Read' is never used.

(unusedFunction)